charts/camunda-platform-8.8/test/integration/testsuites/tests/helper.ts

```diff
@@ -8,7 +8,7 @@ export async function fetchToken(
   config: any,
 ) {
   if (config.authType !== "basic") {
-    const r = await api.post(config.authURL, {
+    let r = await api.post(config.tokenUrl, {
```

Comment:
switching from posting from the authURL to the tokenUrl doesn't make sense.

the auth url is meant to get a code from the authorization server  that you then call the token url from the resource server. to in-place switch from one to the other means it's either completely broken now, or it was completely broken before. And I don't think it was completely broken before.